app/index.js

```diff
@@ -27,15 +26,50 @@ import StorageWindow from './windows/Storage/window'
 // Let's create the main window
 app.mainWindow = null
 
-// A little space for IPFS
+// A little space for IPFS processes
 global.IPFS_PROCESS = null
 global.IPFS_CLIENT = null
 
+// Sets default values for IPFS configurations
+global.IPFS_BINARY_PATH = 'go-ipfs/ipfs'
```

Comment:
I'm not sure if this will work, we need to test the app after building it

Reply:
You might be right! I think we need to re-introduce `app-root-dir`